lib/pages/people/view/person_view.dart

```diff
@@ -0,0 +1,128 @@
+              shape: BoxShape.rectangle,
+              border: new Border.all(
+                  color: Theme.of(context).accentColor, width: 10),
+            ),
+            child: Center(
+              child: Text(person.name,
+                  style: Theme.of(context).textTheme.headline6),
+            ),
+          ),
+          Padding(
+            padding: const EdgeInsets.all(8.0),
+            child: Card(
+              child: Padding(
+                padding: const EdgeInsets.all(8.0),
+                child: IntrinsicHeight(
+                  child: Row(
+                    children: [
+                      Padding(
+                        padding: const EdgeInsets.only(left: 8.0, right: 16.0),
+                        child: person.photo != null
+                            ? CircleAvatar(
+                                maxRadius: 50,
+                                backgroundImage:
+                                    CachedNetworkImageProvider(person.photo),
+                              )
+                            : CircleAvatar(
+                                radius: 50,
+                                child: Icon(
+                                  Icons.person,
+                                  size: 50,
+                                ),
+                              ),
+                      ),
+                      Column(
+                        crossAxisAlignment: CrossAxisAlignment.start,
+                        children: [
+                          RichText(
+                            maxLines: 2,
+                            softWrap: true,
+                            text: TextSpan(
+                                style: Theme.of(context).textTheme.bodyText1,
+                                children: [
+                                  WidgetSpan(
+                                      child: Padding(
+                                    padding: const EdgeInsets.only(right: 4.0),
+                                    child: Icon(Icons.email),
+                                  )),
+                                  TextSpan(text: person.email ?? '-'),
+                                ]),
+                          ),
+                          SizedBox(height: 8),
+                          RichText(
+                            maxLines: 2,
+                            softWrap: true,
+                            text: TextSpan(
+                                style: Theme.of(context).textTheme.bodyText1,
+                                children: [
+                                  WidgetSpan(
+                                      child: Padding(
+                                    padding: const EdgeInsets.only(right: 4.0),
+                                    child: Icon(Icons.phone),
+                                  )),
+                                  TextSpan(text: person.phone ?? '-'),
+                                ]),
+                          ),
```

Comment:
PR #22 was merged, you can now replace all of these with `IconText` widgets. That should fix both the overflow and text theme issues.